host/src/pack/dpkg/opt/cfc/mwc/bin/is-wakelock-active

```diff
@@ -0,0 +1,12 @@
+#! /bin/sh
+if adb get-state 1>/dev/null 2>&1
+then
+        state=$(adb shell dumpsys power | grep -oE 'SCREEN_BRIGHT_WAKE_LOCK')
```

Comment:
better to check if android guest is active. 
If no android app/windows is active. We don't need to do the adb check.

Reply:
Line 2 checks if guest is active or not using adb. 
If not adb, we need to use qemu process for checking the activeness of guest.